website/_sass/custom/custom.scss

```diff
@@ -0,0 +1,24 @@
+// We should override $nav-width(-md), however this breaks code highlighting and other styles.
+// This seems to be an issue for following the recommended approach.
+// See https://github.com/just-the-docs/just-the-docs/issues/982.
+@include mq(lg) {
+    .side-bar {
+        min-width: 400px;
+    }
+
+    .site-nav, .site-header {
+        width: 400px;
+    }
+}
+
+// Add support for external anchor icons.
+.external > svg {
+    width: 1rem;
+    vertical-align: text-bottom;
+}
+
+footer > img#logo {
+    width: 2rem;
+    margin: 0 auto;
+    display: block;
+}
```

Comment:
Indeed, we're setting both left and right margin to `auto` which means we're centering it.

It does require `display: block;`. Otherwise `margin: 0 auto;` won't work. Even if we're lucky that another style already forces that, we're explicitly requiring this to make the centering work. (Though if you remove, it doesn't actually center so we're not lucky in this instance)

Reply:
> Though if you remove, it doesn't actually center so we're not lucky in this instance

I could swear that I tried this... but you're right :+1: